### ISSUE TITLE
Custom position of chart axes

### ISSUE THREAD

**Issue by pavelgruba** [DevExpress/DevExtreme#7263]
# The Problem

Chart axes are always rendered on the chart's edges. This limitation impairs the chart's readability in scientific data analysis.

# The Proposed Solution

We are going to give the chart axes the capability to intersect at any value.

The axes already have the `position` option that accepts `left`, `right`, `top`, or `bottom`. After the implementation, this option will also accept a number that specifies the intersection value. In the following code, the axes intersect at zero:

```js
let chartOptions = {
    argumentAxis: {
        position: 0
    },
    valueAxis: {
        position: 0
    }
};
```

If the intersection value is outside the visual range, the axis will be rendered at the nearest edge. 
<img width="600" alt="Screenshot 2019-03-13 at 15 19 12" src="https://user-images.githubusercontent.com/19471748/54278451-6b147780-45a3-11e9-9e26-08e3d43b939d.png">

# We Need Your Feedback

## Take a Quick Poll

[Do you find the capability to relocate axes useful?](https://docs.google.com/forms/d/e/1FAIpQLSckAvZ75THE3-33wnkKF7eSINUg7FWSnG6ABRoEKa-leMpPuw/viewform?usp=sf_link)

**Comment by alexanderPolosatov** [comment]
This feature is now available in the recent [20.1-Sprint 5](https://github.com/DevExpress/DevExtreme/releases/tag/20.1.1-sprint.5) pre-release version. You can try it online in the [Angular](https://codesandbox.io/s/axis-custom-position-devextreme-charts-xxnis), [React](https://codesandbox.io/s/axis-custom-position-devextreme-charts-oktoj), [VueJS](https://codesandbox.io/s/axis-custom-position-devextreme-charts-68zv1), and [jQuery](https://codepen.io/p_alexander/pen/KKpQLBm) sandboxes.

**Comment by alexanderPolosatov** [comment]
Thanks to everybody who gave us early feedback on this feature. The feature is now available in the [v20.1 release](https://github.com/DevExpress/DevExtreme/releases/tag/20.1.3). I'm closing this thread. In the case of bugs or questions, feel free to create new [GitHub issues](https://github.com/DevExpress/DevExtreme/issues) or tickets in our [Support Center](https://www.devexpress.com/Support/Center/).
